README.rst

```diff
@@ -37,11 +36,43 @@ One Time Setup
   # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
   mkvirtualenv -p python3.8 edx-cookiecutters
 
+2. Create a cookiecutter Repository
+--------------------------------
+
+These instructions assume you have cloned this repository and are currently in its head dir. You will need a virtualenv for running the cookiecutter. You can discard it once the cookiecutter has made your new repo.
+
+Commands::
+
+    $ make requirements  # from inside edx-cookiecutter repo
+    # Replace <OUTPUT-DIRECTORY> with the base directory; your new directory will go inside.
+    # Replace <COOKIECUTTER-NAME> with one of the available cookiecutters documented above.
+    $ cookiecutter -o <OUTPUT-DIRECTORY> <COOKIECUTTER-NAME>
+
+3. TODOs after creating cookiecutter
+-----------------------------------
+
+- Modify project README
+- Modify project docs/decisions/0001-purpose-of-this-repo.rst ADR
+
+Updating cookiecutters
+**********************
+
+If you find anything that is outdated in the cookiecutters in this repository, please create a PR with updates.
+
+
+.. Note:: Some of the cookiecutters in this repository use the layered cookiecutter approach. If you are modifying these, please see "Local Debugging of the layered cookiecutters" section below.
+
 
-Every time you develop something in this repo
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Directions for contributing to this repository
+----------------------------------------------
 .. code-block::
 
+  # Clone the repository
+  git clone git@github.com:edx/edx-cookiecutters.git
+  cd edx-cookiecutters
+
+  # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
+  mkvirtualenv -p python3.8 edx-cookiecutters
   # Activate the virtualenv
   workon edx-cookiecutters
```

Comment:
I think I'd prefer we got rid of any explicit CLI instructions on how to use virtualenvs, since people use different tools.